src/views/Compendium/Compendium.jsx

```diff
@@ -1,84 +1,94 @@
-import {useState, useEffect} from 'react';
-import PokemonList from '../../components/PokemonList/PokemonList';
+import { useState, useEffect } from "react";
+import PokemonList from "../../components/PokemonList/PokemonList";
 import {
   fetchFilteredPokemon,
   fetchPokemon,
   fetchSearchPokemon,
   fetchTypes,
-} from '../../services/pokemon';
-import './Compendium.css';
-import Controls from '../../components/Controls/Controls';
-import pokeball from '../../assets/pokeball.png';
+} from "../../services/pokemon";
+import "./Compendium.css";
+import Controls from "../../components/Controls/Controls";
+import pokeball from "../../assets/pokeball.png";
+// import { getByPlaceholderText } from "@testing-library/dom";
 
 export default function Compendium() {
   const [loading, setLoading] = useState(true);
   const [pokemons, setPokemons] = useState([]);
-  const [searchName, setSearchName] = useState('');
+  const [searchName, setSearchName] = useState("");
   const [types, setTypes] = useState([]);
-  const [selectedType, setSelectedType] = useState('all');
+  const [selectedType, setSelectedType] = useState("all");
 
-  //TODO 😖 help!
+  // TODO 😖 help!
   // if (pokemons.count !== 0) {
-  //   useEffect(() => {
-  //     const getPokemon = async () => {
-  //       const pokemonList = await fetchPokemon();
-  //        this.setState({pokemons: pokemonList});
-  //       setLoading(false);
-  //     };
-  //     getPokemon();
-  //   }, [pokemons]);
+  useEffect(() => {
+    // const getPokemon = async () => {
```

Comment:
The commented line and this line do the same thing so you could've gone with the original or just gotten rid of the commented out code on line 24

Reply:
Thank you for pointing that out!